cucumber/rotators/features/support/rotator_helpers.rb

```diff
@@ -52,13 +51,13 @@ def pg_history_after_rotation(var_name:, db_user:, orig_pw:)
   # This represents a rotating postgres password across time -- a changing
   # entity with a current_value.
   # 
-  # The "value" of this entity only exiss when the actual db password matches
+  # The "value" of this entity only exists when the actual db password matches
   # the password in Conjur.  During the ephemeral moments when they're out of
   # sync, or when either one of the passwords is not available, the
   # `PgRotatingPassword` is considered to be `nil`.
   #
   # This avoids possible race conditions with the actual rotation thread --
-  # it's possible we could "reading" here at the same time the rotation process
+  # its possible we could "reading" here at the same time the rotation process
```

Comment:
The original is correct. "It is possible we"

Reply:
Fixed